Examples/MAX32690/Flash/main.c

```diff
@@ -241,8 +239,11 @@ int main(void)
     int err = 0;
 
     printf("\n\n***** Flash Control Example *****\n");
+#if defined(EvKit_V1)
     printf("Press Push Button 1 (PB1/SW1) to continue...\n\n");
-
+#else
+    printf("Press Push Button 1 (PB1/SW3) to continue...\n\n");
```

Comment:
In past i added PB_IsPressedAny function to able to eliminate this type comment, 
https://github.com/Analog-Devices-MSDK/msdk/blob/main/Libraries/MiscDrivers/PushButton/pb.c#L133C1-L133C1


See usage:
https://github.com/Analog-Devices-MSDK/msdk/blob/main/Examples/MAX32672/Comparator/main.c#L79

It can simplify the startup messages too,
Due to it works in polling mode, the led delay can be decreased...

Reply:
Done.